pkg/cagent/rpm/cagent.spec

```diff
@@ -0,0 +1,52 @@
+%global debug_package %{nil}
+
+Name: cagent
```

Comment:
I haven't used cagent; does it use the CLI and/or daemon? We could add a "weak" dependency if it does, e.g. as we did for the model CLI (although that's more clearly a CLI plugin);

```
Enhances: docker-ce-cli
```

Reply:
> I haven't used cagent

😱 

It's not a cli plugin no

Reply:
@rumpl does it have any dependency / recommended installation w.r.t. "if you use this, you should also have `docker` or `docker-cli` installed? (mostly trying to see if we need to define some relation between the packages, or it it should be considered fully separate from our other offerings)

Reply:
A recommended would be Docker Desktop actually, for pushing and pulling agents.